.eslintrc.js

```diff
@@ -1,10 +1,17 @@
 module.exports = {
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    requireConfigFile: false,
+    babelOptions: {
+      presets: ['@babel/preset-react'],
```

Comment:
I am pretty sure we want to just use `next/babel` (or something) here

Reply:
```
"presets": ["next/babel"]
```
https://nextjs.org/docs/advanced-features/customizing-babel-config

Reply:
Didn't realize Next had a babel config, thanks!